packages/mc-dev-authentication/middlewares/create-mc-dev-authentication-middleware.js

```diff
@@ -0,0 +1,84 @@
+const path = require('path');
+const pug = require('pug');
+const { logout } = require('../routes');
+
+const compileLoginView = pug.compileFile(
+  path.join(__dirname, '../views/login.pug')
+);
+const compileLogoutView = pug.compileFile(
+  path.join(__dirname, '../views/logout.pug')
+);
+
+function createMcDevAuthenticationMiddleware(applicationConfig) {
```

Comment:
This is an "aggregated" middleware for handling login/logout requests from developing against the local MC API.

This is only useful for internal teams.